texar/torch/data/tokenizers/gpt2_tokenizer.py

```diff
@@ -70,6 +70,24 @@ class GPT2Tokenizer(TokenizerBase, PretrainedGPT2Mixin):
         'vocab_file': 'encoder.json',
         'merges_file': 'vocab.bpe',
     }
+    _VOCAB_FILE_MAP = {
+        'vocab_file': {
+            'gpt2-small': 'encoder.json',
+            'gpt2-medium': 'encoder.json',
+            'gpt2-large': 'encoder.json',
+            'gpt2-xl': 'encoder.json',
+            '117M': 'encoder.json',
+            '345M': 'encoder.json',
+        },
+        'merges_file': {
```

Comment:
what does `merges` mean here?

Reply:
BPE generates subword tokens by merging the most frequent consecutive byte pairs. This file contains all the merged byte pairs.